ws-nextjs-app/pages/[service]/send/[id]/styles.ts

```diff
@@ -37,22 +36,9 @@ export default {
         paddingBottom: `${spacings.QUADRUPLE}rem`,
       },
     }),
-  mainContent: ({ spacings, fontVariants, palette, mq }: Theme) =>
+  mainContent: ({ spacings, palette, mq }: Theme) =>
     css({
-      a: {
-        ...fontVariants.sansBold,
-        color: palette.EBON,
-        borderBottom: `${pixelsToRem(1)}rem solid ${palette.POSTBOX}`,
-        textDecoration: 'none',
-        '&:visited': {
-          color: palette.METAL,
-          borderBottom: `${pixelsToRem(1)}rem solid ${palette.METAL}`,
-        },
-        '&:focus, &:hover': {
-          borderBottom: `${pixelsToRem(2)}rem solid ${palette.POSTBOX}`,
-          color: palette.POSTBOX,
-        },
-      },
+      a: getInlineLinkStyles(palette),
```

Comment:
I'm not sure if we need this here as the `description` and `privacyNotice` classes already define this CSS: https://github.com/bbc/simorgh/blob/0141e0d7920950cd908fe7593a7056a991055c07/ws-nextjs-app/pages/%5Bservice%5D/send/%5Bid%5D/Form/styles.ts#L20-L41